server/sandbox_remove.go

```diff
@@ -18,7 +18,13 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 	logrus.Debugf("RemovePodSandboxRequest %+v", req)
 	sb, err := s.getPodSandboxFromRequest(req)
 	if err != nil {
-		return nil, err
+		if err == errSandboxIDEmpty {
+			return nil, err
+		}
+
+		resp := &pb.RemovePodSandboxResponse{}
+		logrus.Warnf("could not get sandbox: %v", err)
```

Comment:
nit: make log more clear, e.g. `Couldn't get sandbox with ID %q, assuming it has already been removed`